packages/contracts-bedrock/contracts/L1/ResourceMetering.sol

```diff
@@ -104,20 +98,16 @@ abstract contract ResourceMetering is Initializable {
                 // Update the base fee by repeatedly applying the exponent 1-(1/change_denominator)
                 // blockDiff - 1 times. Simulates multiple empty blocks. Clamp the resulting value
                 // between min and max.
-                newBaseFee = SignedMath.min(
```

Comment:
The new version or the old version?

Reply:
Or both?

Reply:
both

Reply:
What do you think about putting this arithmetic into a library? Then it will be very clean here, also the `private` function added here could be in the library